go.sum

```diff
@@ -13,6 +13,8 @@ github.com/davecgh/go-spew v1.1.1 h1:vj9j/u1bqnvCEfJOwUhtlOARqs3+rkHYY13jYWTU97c
 github.com/davecgh/go-spew v1.1.1/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
 github.com/devcyclehq/go-server-sdk/v2 v2.14.0 h1:Hl1NzvfzuMrtH8EgTQ5o8Nbf/llPxVKftW2AqpU2gCE=
 github.com/devcyclehq/go-server-sdk/v2 v2.14.0/go.mod h1:ya5guuuahBmVjJr4Et1ifKXfXCc5ELsMW0ZmgXHaqm4=
```

Comment:
should these (v2.14.0) references be removed since that version is no longer used?

```suggestion
```

Reply:
Im not sure how this works, this file was auto-generated from me running `go get`

Reply:
I dont think Im supposed to manually modify this file

Reply:
yeah.. shouldn't manually modify, but we should be able to remove references to the old version :) maybe `go mod tidy`?

Reply:
yep, got that to work after some version shenanigans